packages/app/src/systems/Settings/__mocks__/abi.ts

```diff
@@ -0,0 +1,79 @@
+import { AbiService } from '../services';
+
+export const MOCK_ABI_MAP = {
```

Comment:
```suggestion
export const MOCK_ABI_MAP: AbiMap = {
```
would it be useful to do this?